Description: Adds a missing man pages for wsrep_* commands
 Remove this once they are included by upstream.
Author: Ian Gilfillan <ian@mariadb.org>

--- a/man/CMakeLists.txt
+++ b/man/CMakeLists.txt
@@ -22,7 +22,9 @@ SET(MAN1_SERVER innochecksum.1 my_print_
 		mysql_tzinfo_to_sql.1 mysql_upgrade.1 mysql_zap.1
 		mysqld_multi.1 mysqld_safe.1 mysqldumpslow.1 mysqlhotcopy.1
 		mysqltest.1 perror.1 replace.1 resolve_stack_dump.1
-		resolveip.1 mysqlbug.1 tokuftdump.1)
+		resolveip.1 mysqlbug.1 tokuftdump.1
+		wsrep_sst_common.1 wsrep_sst_mysqldump.1 wsrep_sst_rsync.1
+		wsrep_sst_xtrabackup-v2.1 wsrep_sst_xtrabackup.1)
 SET(MAN8_SERVER mysqld.8)
 SET(MAN1_CLIENT msql2mysql.1 mysql.1 mysql_find_rows.1 mysql_waitpid.1
 		mysqlaccess.1 mysqladmin.1 mysqlbinlog.1 mysqlcheck.1
--- /dev/null
+++ b/man/wsrep_sst_common.1
@@ -0,0 +1,16 @@
+'\" t
+.\"
+.TH "\FBWSREP_SST_COMMON\FR" "1" "24 January 2017" "MariaDB 10\&.1" "MariaDB Database System"
+.\" -----------------------------------------------------------------
+.\" * set default formatting
+.\" -----------------------------------------------------------------
+.\" disable hyphenation
+.nh
+.\" disable justification (adjust text to left margin only)
+.ad l
+.SH NAME
+wsrep_sst_common \- common wsrep state transfer binary
+.SH DESCRIPTION
+Use:
+.PP
+For more information, please refer to the MariaDB Knowledge Base, available online at https://mariadb.com/kb/
--- /dev/null
+++ b/man/wsrep_sst_mysqldump.1
@@ -0,0 +1,16 @@
+'\" t
+.\"
+.TH "\FBWSREP_SST_COMMON\FR" "1" "24 January 2017" "MariaDB 10\&.1" "MariaDB Database System"
+.\" -----------------------------------------------------------------
+.\" * set default formatting
+.\" -----------------------------------------------------------------
+.\" disable hyphenation
+.nh
+.\" disable justification (adjust text to left margin only)
+.ad l
+.SH NAME
+wsrep_sst_mysqldump \- mysqldump-based state snapshot transfer
+.SH DESCRIPTION
+Use: mysqldump-based state snapshot transfer\.
+.PP
+For more information, please refer to the MariaDB Knowledge Base, available online at https://mariadb.com/kb/
--- /dev/null
+++ b/man/wsrep_sst_rsync.1
@@ -0,0 +1,16 @@
+'\" t
+.\"
+.TH "\FBWSREP_SST_COMMON\FR" "1" "24 January 2017" "MariaDB 10\&.1" "MariaDB Database System"
+.\" -----------------------------------------------------------------
+.\" * set default formatting
+.\" -----------------------------------------------------------------
+.\" disable hyphenation
+.nh
+.\" disable justification (adjust text to left margin only)
+.ad l
+.SH NAME
+wsrep_sst_mysqldump \- rsync-based state snapshot transfer
+.SH DESCRIPTION
+Use: rsync-based state snapshot transfer\.
+.PP
+For more information, please refer to the MariaDB Knowledge Base, available online at https://mariadb.com/kb/
--- /dev/null
+++ b/man/wsrep_sst_xtrabackup-v2.1
@@ -0,0 +1,16 @@
+'\" t
+.\"
+.TH "\FBWSREP_SST_COMMON\FR" "1" "24 January 2017" "MariaDB 10\&.1" "MariaDB Database System"
+.\" -----------------------------------------------------------------
+.\" * set default formatting
+.\" -----------------------------------------------------------------
+.\" disable hyphenation
+.nh
+.\" disable justification (adjust text to left margin only)
+.ad l
+.SH NAME
+wsrep_sst_xtrabackup-v2 \- xtrabackup-v2-based state snapshot transfer
+.SH DESCRIPTION
+Use: xtrabackup-based state snapshot transfer\.
+.PP
+For more information, please refer to the MariaDB Knowledge Base, available online at https://mariadb.com/kb/
--- /dev/null
+++ b/man/wsrep_sst_xtrabackup.1
@@ -0,0 +1,16 @@
+'\" t
+.\"
+.TH "\FBWSREP_SST_COMMON\FR" "1" "24 January 2017" "MariaDB 10\&.1" "MariaDB Database System"
+.\" -----------------------------------------------------------------
+.\" * set default formatting
+.\" -----------------------------------------------------------------
+.\" disable hyphenation
+.nh
+.\" disable justification (adjust text to left margin only)
+.ad l
+.SH NAME
+wsrep_sst_xtrabackup \- xtrabackup-based state snapshot transfer
+.SH DESCRIPTION
+Use: xtrabackup-based state snapshot transfer\.
+.PP
+For more information, please refer to the MariaDB Knowledge Base, available online at https://mariadb.com/kb/
